src/InstagramClone.Api/Startup.cs

```diff
@@ -0,0 +1,47 @@
+﻿using InstagramClone.Api.Database;
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api
+{
+    public static class Startup
+    {
+        public static WebApplication InitialIzeApp(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+            ConfigureService(builder);
+            var app = builder.Build();
+            Configure(app);
+            return app;
+        }
+
+        private static void ConfigureService(WebApplicationBuilder builder)
+        {
+            
+            builder.Services.AddDbContext<InstagramCloneDbContext>(options =>
+            {
+                options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnectionString"));
+
+               
+            });
+            builder.Services.AddEndpointsApiExplorer();
+            builder.Services.AddControllers();
+           
+
+        }
+
+        private static void Configure(WebApplication app)
+        {
+            if(app.Environment.IsDevelopment())
+            {
+                //app.UseSwagger(); 
+            }
```

Comment:
what's the purpose of this empty `if` block?

Reply:
I was trying to configure swagger, and I forgot to remove it before pushing

Reply:
okay, I'm not sure why swagger is related to this PR though?